cypress/integration/unit_tests/manage_disaster_add_delete_test.js

```diff
@@ -135,10 +150,64 @@ describe('Add/delete-related tests for manage_disaster.js', () => {
                   'Error: disaster name must be comprised of only ' +
                   'lowercase letters');
 
+          expect(createFolderStub).to.not.be.called;
+          expect(setAclsStub).to.not.be.called;
           name.val('harvey');
           return addDisaster();
         })
-        .then((success) => expect(success).to.be.true);
+        .then((success) => {
+          expect(success).to.be.true;
+          expect(createFolderStub).to.be.calledThrice;
+          expect(setAclsStub).to.be.calledThrice;
+        });
+  });
+
+  /**
+   * Creates an option with the given text, already selected.
+   * @param {string} text
+   * @return {JQuery<HTMLOptionElement>}
+   */
+  function createSelectedOption(text) {
+    return $(document.createElement('option')).val(text).prop('selected', true);
+  }
+
+  it('Error creating EE folder', () => {
+    const id = '2005-summer';
+    const states = [KNOWN_STATE];
+    const errorStub = cy.stub(ErrorLib, 'showError');
+    // Tests don't have permission to create folders, so this will exercise EE
+    // failure mode better than we could fake it.
+    createFolderStub.restore();
```

Comment:
why do you need to restore here?

Reply:
Because it was stubbed in the beforeEach. Expanded comment.

Reply:
Ah sorry got this mixed up with resetHistory